src/api/app/controllers/webui/project_controller.rb

```diff
@@ -894,19 +894,20 @@ def remove_path_from_target
     redirect_to :action => :repositories, :project => @project
   end
 
-  def move_repo(direction)
+  def move_path_up
```

Comment:
Why do you duplicate this? [Don't repeat yourself!](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)